### DISCUSSION TITLE
PBD method

### DISCUSSION THREAD

**Discussion by HNUZhaoXiang** [InteractiveComputerGraphics/SPlisHSPlasH#235] [Q&A]
HI
Is there any specific paper reference for the calculation process of PBD algorithm used in splishsplash

**Comment by janbender** [comment]
SPlisHSPlasH uses our PositionBasedDynamics library which implements several methods from different papers. Take a look at the references here: 

https://github.com/InteractiveComputerGraphics/PositionBasedDynamics#references

In SPlisHSPlasH we mainly use the rigid body dynamics and collision handling as described in these papers: 

- C. Deul, P. Charrier and J. Bender, "Position-Based Rigid Body Dynamics", Computer Animation and Virtual Worlds, 2014
- D. Koschier, C. Deul, M. Brand and J. Bender, "An hp-Adaptive Discretization Algorithm for Signed Distance Field Generation", IEEE Transactions on Visualization and Computer Graphics 23, 2017

**Reply by HNUZhaoXiang** [reply]
ths
